x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/vector/Knn.java

```diff
@@ -209,7 +212,12 @@ protected TypeResolution resolveParams() {
     }
 
     private TypeResolution resolveField() {
-        return isNotNull(field(), sourceText(), FIRST).and(isType(field(), dt -> dt == DENSE_VECTOR, sourceText(), FIRST, "dense_vector"));
+        return isNotNull(field(), sourceText(), FIRST).and(
```

Comment:
should we also update:

https://github.com/elastic/elasticsearch/blob/cb3354856a6a93deeba81b229b4c6518fef077a5/x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/vector/Knn.java#L105

and regenerate the docs?

Reply:
Good catch, done in d8ef10263fe811ffaa9d75a1cd21ec330bc960e7